lib/data/firebase_const.dart

```diff
@@ -2,16 +2,22 @@ const IMAGE_BASE_PATH = "/flamelink/media/sized";
 
 const FLAME_LINK_CONTENT = 'fl_content';
 const FLAME_LINK_SCHEMA = '_fl_meta_.schema';
+const FLAME_LINK_SCHEMA_TYPE = "_fl_meta_.schemaType";
 const FLAME_LINK_ENVIROMENT = '_fl_meta_.env';
 const FLAME_LINK_LOCALE = '_fl_meta_.locale';
```

Comment:
We would need to wrap these const under a common class instead of repeating "FLAME_LINK"